src/ViewHoldingsRecord.test.js

```diff
@@ -397,4 +412,140 @@ describe('ViewHoldingsRecord actions', () => {
       });
     });
   });
+
+describe('Update ownership action item', () => {
+    beforeEach(() => {
+      checkIfUserInMemberTenant.mockClear().mockReturnValue(true);
+    });
+
+    const targetTenantId = 'university';
+    const targetLocationId = 'targetLocationId';
+    const stripes = {
+      ...buildStripes({
+        okapi: { tenant: 'college' },
+        user: {
+          user: {
+            tenants: [{
+              id: 'college',
+              name: 'College',
+            }, {
+              id: 'university',
+              name: 'University',
+            }, {
+              id: 'consortium',
+              name: 'Consortium',
+            }],
+          },
+        },
+      }),
+    };
+
+    describe('when instance is shared', () => {
+      it('should be rendered', async () => {
+        renderViewHoldingsRecord({ stripes, isInstanceShared: true });
+
+        const updateOwnershipBtn = await screen.findByTestId('update-ownership-btn');
+        expect(updateOwnershipBtn).toBeInTheDocument();
```

Comment:
Can getByRole() be used here?
```suggestion
        expect(screen.getByRole('button', { name: 'Update ownership' })).toBeInTheDocument();;
```